config/plugins.yaml

```diff
@@ -244,6 +244,26 @@ triggers:
   only_org_members: true
 
 plugins:
+  falcosecurity/test-infra:
```

Comment:
Can we maintain these keys in alphabetical order, please?

WDYT?

Reply:
Sure, I put it at the top since it had all the comments so people could find it easier. Would you prefer switching back?

Reply:
Fixed